lib/ae_mdw/oracles.ex

```diff
@@ -0,0 +1,118 @@
+defmodule AeMdw.Oracles do
+  @moduledoc """
+  Context module for dealing with Oracles.
+  """
+
+  require AeMdw.Db.Model
+
+  alias AeMdw.Db.Format
+  alias AeMdw.Db.Model
+  alias AeMdw.Mnesia
+  alias AeMdw.Node
+
+  @type cursor :: binary()
+  # TODO: This needs to be an actual type like AeMdw.Db.Oracle.t()
+  @type oracle :: term()
+
+  @limit 10
+
+  @table_active AeMdw.Db.Model.ActiveOracle
+  @table_active_expiration Model.ActiveOracleExpiration
+  @table_inactive AeMdw.Db.Model.InactiveOracle
+  @table_inactive_expiration Model.InactiveOracleExpiration
+
+  @spec fetch_active_oracles(Mnesia.direction(), cursor() | nil) :: {[oracle()], cursor() | nil}
+  def fetch_active_oracles(direction, cursor) do
+    {exp_keys, next_cursor} =
```

Comment:
Does this work for other requests as well? For example, if the client wants to get /txs, it may have to read from different tables.

Reply:
The idea is that this `Oracles` module will only contain oracle-related logic, if there's anything in common with other context functions that might appear in the future we can think of extracting them into something else. Does that sound good?